fonts.css

```diff
@@ -73,3 +73,103 @@
   font-size: 20px;
   line-height: 24px;
 }
+
+.card__header_font {
+ font-family: Montserrat;
+ font-style: normal;
+ font-weight: bold;
+ font-size: 40px;
+ line-height: 41px;
+}
+
+.card__subheading {
+ font-family: Montserrat;
+ font-style: normal;
+ font-weight: bold;
+ font-size: 16px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5642160)